registry-data.service.template error during `make run`

Starting a new vagrant cluster off master just now, I hit this error:

``` console
$ fleetctl --version
fleetctl version 0.5.1
$ make run
Job deis-router@1.service loaded on 42fdf045.../172.17.8.101
Job deis-router@2.service loaded on 4f00985c.../172.17.8.102
Job deis-builder-data.service loaded on fc61de46.../172.17.8.100
Job deis-database-data.service loaded on fc61de46.../172.17.8.100
Job deis-logger-data.service loaded on 4f00985c.../172.17.8.102
Job deis-registry-data.service loaded on 42fdf045.../172.17.8.101
fleetctl --strict-host-key-checking=false load logger/systemd/deis-logger.service cache/systemd/deis-cache.service database/systemd/deis-database.service
Job deis-database.service loaded on fc61de46.../172.17.8.100
Job deis-logger.service loaded on 4f00985c.../172.17.8.102
Job deis-cache.service loaded on 42fdf045.../172.17.8.101
fleetctl --strict-host-key-checking=false load registry/systemd/*.service
Job deis-registry.service loaded on 42fdf045.../172.17.8.101
fleetctl --strict-host-key-checking=false load controller/systemd/*.service
Job deis-controller.service loaded on 4f00985c.../172.17.8.102
fleetctl --strict-host-key-checking=false load builder/systemd/*.service
Job deis-builder.service loaded on fc61de46.../172.17.8.100
Deis components may take a long time to start the first time they are initialized.
Waiting for 1 of 2 deis-routers to start...
fleetctl --strict-host-key-checking=false start -no-block deis-router@1.service; fleetctl --strict-host-key-checking=false start -no-block deis-router@2.service;
Triggered job deis-router@1.service start
Triggered job deis-router@2.service start
Waiting for deis-cache to start...
fleetctl --strict-host-key-checking=false start -no-block logger/systemd/deis-logger.service cache/systemd/deis-cache.service database/systemd/deis-database.service
Triggered job deis-logger.service start
Triggered job deis-cache.service start
Triggered job deis-database.service start
Waiting for deis-registry to start...
fleetctl --strict-host-key-checking=false start -no-block registry/systemd/*
Error starting jobs: unable to find Job(deis-registry-data.service.template.service)
make: *** [start] Error 1
```

There is a deis-registry-data.service.template.service defined in fleet at this point.

--- a/Makefile
+++ b/Makefile
@@ -97,7 +97,7 @@ start: check-fleet start-warning start-routers
 
 	@# registry
 	$(call echo_yellow,"Waiting for deis-registry to start...")
-	$(FLEETCTL) start -no-block registry/systemd/*
+	$(FLEETCTL) start -no-block registry/systemd/*.service
 	@until $(FLEETCTL) $(list-units) | egrep -q "deis-registry\..+(running|failed)"; \
 		do sleep 2; \
 			printf "\033[0;33mStatus:\033[0m "; $(FLEETCTL) $(list-units) | \
@@ -108,7 +108,7 @@ start: check-fleet start-warning start-routers
 
 	@# controller
 	$(call echo_yellow,"Waiting for deis-controller to start...")
-	$(FLEETCTL) start -no-block controller/systemd/*
+	$(FLEETCTL) start -no-block controller/systemd/*.service
 	@until $(FLEETCTL) $(list-units) | egrep -q "deis-controller\..+(running|failed)"; \
 		do sleep 2; \
 			printf "\033[0;33mStatus:\033[0m "; $(FLEETCTL) $(list-units) | \

